src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/out-string/out-string.cs

```diff
@@ -45,6 +45,25 @@ public int Width
         }
 
         private Nullable<int> _width = null;
+
+        /// <summary>
+        /// False to add a newline to the end of the output string, true if not.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter NoNewline
+        {
+            get
+            {
+                return _suppressNewline;
+            }
+            set
+            {
+                _suppressNewline = value;
+            }
```

Comment:
Please use one line formatting as above.

Reply:
Thanks for pointing that out. Done!